supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala

```diff
@@ -166,6 +164,27 @@ class JesAsyncBackendJobExecutionActor(override val jobDescriptor: BackendJobDes
     MetadataJobKey(jobDescriptorKey.call.fullyQualifiedName, jobDescriptorKey.index, jobDescriptorKey.attempt)
   }
 
+  override def receive: Receive = LoggingReceive {
+    case AbortJobCommand =>
+      serviceRegistryActor ! GetMetadataQueryAction(MetadataQuery.forKey(metadataKey("jobId")))
+    case MetadataLookupResponse(query, events) if query.key.contains("jobId") =>
+      /** Currently this actor can only get a MetadataLookupResponse when requesting a
+        * job ID for abort.  If this actor becomes more complex in the future, then this
+        * should probably be received in an FSM state.
+        */
+      events.headOption foreach { event =>
+        val jobId = event.value.value
+        log.info(s"$tag Aborting $jobId")
+        Try(Run(jobId, jobDescriptor, genomicsInterface).abort()) match {
+          case Success(_) => log.info(s"$tag Aborted $jobId")
+          case Failure(ex) => log.warning(s"$tag Failed to abort $jobId: ${ex.getMessage}")
+        }
+      }
+      context.parent ! AbortedResponse(jobDescriptor.key)
+      context.stop(self)
+    case message => super.receive(message)
```

Comment:
Don't fix now, but Miguel pointed out we should probably use a slightly different syntax in the future, using `PartialFunction.orElse`. Please drop a `PBE` todo here, and we can (maybe) fix it later if it's an issue.